keyboards/massdrop/alt/keymaps/ewersp/keymap.c

```diff
@@ -0,0 +1,200 @@
+        KC_TAB,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_END,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, KC_HOME, _______, KC_END
+    )
+};
+
+// If the super alt layer is the active layer
+bool super_alt_layer_active = false;
+
+// If we need to unregister alt when leaving the super alt layer
+bool need_to_unregister_alt = false;
+
+#define MODS_SHIFT  (get_mods() & MOD_BIT(KC_LSHIFT) || get_mods() & MOD_BIT(KC_RSHIFT))
+#define MODS_CTRL  (get_mods() & MOD_BIT(KC_LCTL) || get_mods() & MOD_BIT(KC_RCTRL))
+#define MODS_ALT  (get_mods() & MOD_BIT(KC_LALT) || get_mods() & MOD_BIT(KC_RALT))
+
+// Taken from 'g_led_config' in config_led.c
+#define CAPS_LOCK_LED_ID 30
+
+// This runs every matrix scan (every 'frame')
+void rgb_matrix_indicators_user(void) {
+    led_flags_t flags = rgb_matrix_get_flags();
+
+    // If we're in either keylight or underglow modes (but not both simultaneously)
+    if (HAS_FLAGS(flags, LED_FLAG_KEYLIGHT) != HAS_FLAGS(flags, LED_FLAG_UNDERGLOW)) {
+
+        // This fixes a bug where the caps lock LED will flicker when toggled in either keylight or underglow modes
+        if (host_keyboard_leds() & (1 << USB_LED_CAPS_LOCK)) {
+            rgb_matrix_set_color(CAPS_LOCK_LED_ID, RGB_WHITE);
+        } else {
+            rgb_matrix_set_color(CAPS_LOCK_LED_ID, 0, 0, 0);
+        }
+    }
+}
+
+// This runs code every time that the layers get changed
+layer_state_t layer_state_set_user(layer_state_t state) {
+    switch (get_highest_layer(state)) {
+        case DEF:
+            // When returning to the default layer, check if we need to unregister the left alt key
+            if (super_alt_layer_active && need_to_unregister_alt) {
+                unregister_code(KC_LALT);
+            }
+
+            super_alt_layer_active = false;
+            need_to_unregister_alt = false;
+            break;
+        case SUPR:
+            super_alt_layer_active = true;
+            break;
+    }
+    return state;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    static uint32_t key_timer;
+
+    switch (keycode) {
+        // These are the keys we want to 'fall though' and behave as usual when pressed with the alt modifier
+        case KC_A: case KC_B: case KC_C: case KC_D: case KC_E:
+        case KC_F: case KC_G: case KC_H: case KC_I: case KC_J:
+        case KC_K: case KC_L: case KC_M: case KC_N: case KC_O:
+        case KC_P: case KC_Q: case KC_R: case KC_S: case KC_T:
+        case KC_U: case KC_V: case KC_W: case KC_X: case KC_Y: case KC_Z:
```

Comment:
```suggestion
        case KC_A ... KC_Z:
```

Reply:
Nice, didn't know that syntax was valid, updated.